File: Python/crescimento_populacional.py
t=int(input())
cont2= 1   
while cont2<=t:
    cont = 1
    pa=int(input())
    pb=int(input())
    g1=float(input())
    g2=float(input())
    
    pa = pa + int((pa * (g1/100)))
    if g2 == 0:
        pb = pb
        while pa <= pb and cont <= 100:
            pa += int((pa * (g1/100)))
            cont+=1
    if g2 > 0:
        pb = pb + int(pb * (g2/100))
        while pa <= pb and cont <= 100:
            pa += int(pa * (g1/100))
            pb += int(pb * (g2/100))
            cont+=1
    if cont > 100:
        print("Mais de 1 seculo.")
        break
    else:
        print(f"{cont} anos.")
    cont2+=1

File: C/crescimento_populacional.cpp
#include<stdio.h>

int main(void){
	int pa, pb,t;
	int cont2 = 1, cont = 0;
	double g1, g2;
	
	scanf("%d", &t);
	while(cont2 <= t){
		cont = 0;
		scanf("%d %d %lf %lf", &pa, &pb, &g1, &g2);
		while(pb >= pa ){
            pa += int(pa * (g1/100));
            pb += int(pb * (g2/100));
			cont++;
			if(cont > 100) break;			
		}
		if(cont > 100){
			printf("Mais de 1 seculo.\n");
		}
		else{
			printf("%d anos.\n",cont);
		}
			
		cont2++;
	}
	
	return 0;
}
